solr/core/src/test/org/apache/solr/update/processor/ParsingFieldUpdateProcessorsTest.java

```diff
@@ -992,6 +992,14 @@ public void testLenient() throws IOException {
     assertParsedDate("Friday Oct 7 13:14:15 2005", Date.from(inst20051007131415()), "parse-date-patterns-default-config");
   }
 
+  public void testRfc2616() throws Exception {
```

Comment:
Should be named testAsctime since that's the format of this input you are testing.  When I mentioned RFC2616, that is a spec that in turn refers to multiple formats (including asctime).

Also, like I pointed out, need to test with 2 spaces left of a single digit day.